pkgs/by-name/vc/vcpkg/package.nix

```diff
@@ -65,7 +65,7 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     '';
 
   meta = {
-    description = "C++ Library Manager";
+    description = "C++ Library Manager for Windows, Linux, and MacOS";
```

Comment:
The official capitalization is “macOS” ([source](https://apple.com/macos)).

```suggestion
    description = "C++ Library Manager for Windows, Linux, and macOS";
```

Reply:
I simply copy pasted the description they put on their GitHub repo.

Reply:
OK. [I just submitted an upstream issue for this problem.](https://github.com/microsoft/vcpkg/issues/40433)